tests/integration/security/sds_vault_flow/sds_vault_flow_test.go

```diff
@@ -0,0 +1,151 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sds_vault_test
+
+import (
+	"path"
+	"testing"
+	"time"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/apps"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/util/connection"
+	"istio.io/istio/pkg/test/util/retry"
+)
+
+var (
+	inst istio.Instance
+)
+
+const (
+	tlsRootCert = "-----BEGIN CERTIFICATE-----\\\\nMIIC3jCCAcagAwIBAgIRAIcSFH1jneS0XPz5r2Q" +
```

Comment:
suggest to change to `---BEGIN CERTIFICATE` for better readability

Reply:
I tried the backtick multiline string but it does not work since 1)linter does not allow long line. 2) if the long line is splitted into multiple lines using backtick multiline string, it adds a newline char for each line to the original line, which makes the certificate unrecognizable to the recipient. I tried multiple ways to format the certificate and the only one works is the one in the PR. 
The example program at https://play.golang.org/p/nnSaeukrSRy demonstrates that if a long line is splitted into multiple lines using backtick multiline string, it adds a newline char for each line to the original line.